packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -832,47 +861,30 @@ export class StatsAnalyzer extends EventsScope {
     LoggerProxy.logger.trace('StatsAnalyzer:index#getStatsAndParse --> Collecting Stats');
 
     return this.mediaConnection.getTransceiverStats().then((transceiverStats) => {
-      this.filterAndParseGetStatsResults(
-        transceiverStats.video.sender,
-        STATS.VIDEO_CORRELATE,
-        true
+      transceiverStats.video.senders.forEach((sender, i) =>
```

Comment:
see my previous comment about local tracks - I think here we should just throw if we have more than 1 sender and hardcode everything for now to work with just 1 video sender (so use mediaType as "video-send")
and same for audio